README.md

```diff
@@ -40,22 +40,20 @@ RisingLight developers are active in a variety of places:
 
 ### Slack Channel
 
-The RisingLight project is currently part of the RisingWave community. You may join RisingWave community with the [invitation link](https://join.slack.com/t/risingwave-community/shared_invite/zt-120rft0mr-d8uGk3d~NZiZAQWPnElOfw). If the Slack link expires (which normally should not happen), please create an issue :)
+The RisingLight Slack channel is currently hosted in the RisingWave community. You may step in the RisingWave community with the [invitation link](https://join.slack.com/t/risingwave-community/shared_invite/zt-120rft0mr-d8uGk3d~NZiZAQWPnElOfw), and then join the following channels:
 
-After that, you may join `risinglight-chinese` and `risinglight-english` channel.
-
-* [risinglight-chinese](https://risingwave-community.slack.com/archives/C02UZDEE4AC)
 * [risinglight-english](https://risingwave-community.slack.com/archives/C030SJRDT4J)
+* [risinglight-chinese](https://risingwave-community.slack.com/archives/C02UZDEE4AC)
 
-### WeChat Group
+### Other Messaging Apps
 
-If you want to join our WeChat group, please send an email to `hezi at singularity-data.com` with your WeChat ID. We will invite you to the WeChat group.
+If you want to join the communication group in messaging apps like Discord, Telegram, or WeChat, please send an email to `contact at singularity-data.com` with your user ID. We will then invite you to the group.
```

Comment:
This part doesn't look clear enough. I would prefer stating clearly that we have a WeChat group, and categorize it under "other messaging apps".

Reply:
I tweaked a little bit more on this sentence to indicate we are using (or will use) all these apps. If we simply state that we are active in a certain venue (e.g., WeChat), it actually gives people the impression that we do not and will not talk in other venues.